### PR TITLE
[ARM64/Unix] Enable FEATURE_EVENT_TRACE

### LINKED ISSUE #14148: [Arm64/Unix] Enable FEATURE_EVENT_TRACE
tracing/eventpipesmoke/eventpipesmoke/eventpipesmoke.sh

```
./corerun eventpipesmoke.exe
Start: Enable tracing.
End: Enable tracing.

Start: Allocation.
End: Allocation.

Start: Disable tracing.
End: Disable tracing.

Created 387 bytes of data
Expected: 100
Actual: 0
```

@dotnet/arm64-contrib

### PR DESCRIPTION
**Pull request by sdmaclea**
Fixes #14148 

@janvorli @brianrob PTAL
cc @dotnet/arm64-contrib

### CODE REVIEW

File: clrfeatures.cmake [hunk 1]
```diff
@@ -18,6 +18,8 @@ if(NOT DEFINED FEATURE_EVENT_TRACE)
       set(FEATURE_EVENT_TRACE 1)
     elseif(CLR_CMAKE_TARGET_ARCH_ARM)
       set(FEATURE_EVENT_TRACE 1)
+    elseif(CLR_CMAKE_TARGET_ARCH_ARM64)
+      set(FEATURE_EVENT_TRACE 1)
```
Comment (sdmaclea) [inline clrfeatures.cmake]: This is now enable for all platforms except x86.

**Comment by sdmaclea** [comment]
@dotnet-bot test Windows_NT arm64 Cross Checked Build and Test

**Comment by nategraf** [comment]
@dotnet-bot test Ubuntu pri1 please

**Comment by sdmaclea** [comment]
@janvorli I verified the required package was present for the arm64 rootfs.  It has been present for > 6 months

**Comment by janvorli** [comment]
@sdmaclea great, thank you for checking

**Comment by sdmaclea** [comment]
It doesn't look like the remaining two CI tests will trigger and run.  This should be safe to merge

**Comment by sdmaclea** [comment]
@janvorli @brianrob If you ignore the x86 CI issues, this can be merged.

**Pull request merged**